resqpy/organize/BoundaryFeature.py

```diff
@@ -0,0 +1,36 @@
+from .organize_functions import alias_for_attribute, equivalent_extra_metadata
```

Comment:
the rest of the resqpy code base uses lowercase file names with underscores – I'd prefer us to stick to that protocol, so boundary_feature.py instead of BoundaryFeature.py

Reply:
done